backend/src/main/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolver.java

```diff
@@ -46,6 +48,22 @@ public AddTestResultResponse addTestResultNew(
         deviceSpecimenTypeId, TestResult.valueOf(result), patientID, dateTested);
   }
 
+  public AddTestResultResponse addTestResultMultiplex(
+      String deviceID,
+      UUID deviceSpecimenType,
+      List<MultiplexTestResult> results,
+      UUID patientID,
+      Date dateTested)
+      throws NumberParseException {
+    UUID deviceSpecimenTypeId =
+        deviceSpecimenType == null
+            ? _dts.getFirstDeviceSpecimenTypeForDeviceTypeId(UUID.fromString(deviceID))
+                .getInternalId()
+            : deviceSpecimenType;
```

Comment:
Outside the scope for this PR, but I'd love to see this deviceSpecimenType logic broken into a helper method at some point (leaving this comment so that I will hopefully remember to make a ticket for it 😄 )